passivetotal.json

```diff
@@ -4,16 +4,16 @@
     "description": "This app implements investigative actions by integrating with the PassiveTotal cloud reputation service",
     "type": "reputation",
     "main_module": "passivetotal_connector.py",
-    "app_version": "2.1.2",
+    "app_version": "2.2.0",
     "utctime_updated": "2021-12-01T22:49:04.000000Z",
     "product_vendor": "PassiveTotal",
     "product_name": "PassiveTotal",
     "product_version_regex": ".*",
-    "min_phantom_version": "4.9.39220",
+    "min_phantom_version": "5.0.0",
     "fips_compliant": true,
     "python_version": "3",
     "latest_tested_versions": [
-        "Cloud v2 tested 2-21-2020"
```

Comment:
As we are not performing full QA , please do not update latest tested version

Reply:
@ishans-crest  Done.